extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/Job.java

```diff
@@ -66,7 +72,7 @@ public static void schedule(String thingUID, AstroThingHandler astroHandler, Job
             logger.error("{}", ex.getMessage(), ex);
         }
     }
-
+    
```

Comment:
I could be wrong -- but I am pretty sure, the formatting rules and the save actions contain the removal of trailing white spaces.
Please check it.
I re-installed the IDE using the "latest release", so Oxygen RC yesterday, too.
I hope it will not contain some regressions about white space handling or save actions.